.github/workflows/build-push-auto.yml

```diff
@@ -154,7 +156,7 @@ jobs:
           files: |
             ./docker-bake.hcl
             ${{ steps.meta.outputs.bake-file }}
-          push: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/main') || inputs.force-push }}
+          push: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/main') || inputs.force-push || github.event_name == 'schedule' }}
```

Comment:
Hmm. this will trigger on *any* schedule run, which is possibly problematic (schedules run against main, generally, but don't have to). I've updated this in 22134acb693ba0a046cbfa1449892cec2cb73913 and 6ff1855a9260e6c6f795b211b41c34fc6afa4a4c to use "if (push OR schedule) AND (is main branch)" just to be safe.